lib/texture.c

```diff
@@ -884,8 +885,13 @@ ktxTexture_GetImageSize(ktxTexture* This, ktx_uint32_t level)
     assert (This != NULL);
 
     formatInfo = &((ktxTextureInt*)This)->formatInfo;
-    blockCount.x = MAX(1, (This->baseWidth / formatInfo->blockWidth)  >> level);
-    blockCount.y = MAX(1, (This->baseHeight / formatInfo->blockHeight)  >> level);
+
+    float levelWidth  = This->baseWidth >> level;
+    float levelHeight = This->baseHeight >> level;
+    int x = ceilf(levelWidth / formatInfo->blockWidth);
+    int y = ceilf(levelHeight / formatInfo->blockHeight);
+    blockCount.x = MAX(1, x);
+    blockCount.y = MAX(1, y);
     blockSizeInBytes = formatInfo->blockSizeInBits / 8;
```

Comment:
Have you compiled this with Visual Studio? I got some warnings from VS2017 about truncation. Some casts are needed to avoid these. Please update this PR. Something like the following should work
```
    float levelWidth  = (float)(This->baseWidth >> level);
    float levelHeight = (float)(This->baseHeight >> level);
    blockCount.x = (ktx_uint32_t)ceilf(levelWidth / formatInfo->blockWidth);
    blockCount.y = (ktx_uint32_t)ceilf(levelHeight / formatInfo->blockHeight);
    blockCount.x = MAX(1, blockCount.x);
    blockCount.y = MAX(1, blockCount.y);
```